build.sh

```diff
@@ -33,7 +36,7 @@ main () {
     shift
     done
 
-    if [[ -z "${buildLabel}" || -z "${fullDownloadUrl}" || -z "${version}" ]]; then
+    if [[ -z "${buildLabel}" || -z "${fullDownloadUrl}" || -z "${version}" || -z "${kernelDownloadUrl}" ]]; then
       echo "Error: buildLabel, fullDownloadUrl, and version are required flags"
```

Comment:
for completeness you could add to the error message that `kernelDownloadUrl` is also needed.